### DISCUSSION TITLE
How can we combine different plugins into the same trial ?

### DISCUSSION THREAD

**Discussion by MatthieuFra** [jspsych/jsPsych#2367] [Q&A] [answered]
Hi ! 

I'm building an experiment using the BWS method (best-worst scaling) for audio experiment. 
In order to do so, I need both Likert Scale, and audio, and a button at the end to pursuit the experiment.
Is there any option to use multiple plugins, but not sequentially ? 

I've been reading pretty much all the internet but I couldn't find any clue, so I try to ask here, even if I suppose it's not possible unless I Create a "meta-plugin" modifying several plugins ? 

Thanks a lot,

**Comment by jodeleeuw** [comment, marked as answer]
Hi @MatthieuFra,

Plugins cannot be layered on top of each other. You would need to create a new plugin to implement the task.

It sounds like the `audio-slider-response` plugin might be pretty close to what you need. You can set labels on the slider to make it behave like a Likert scale, and there is a button to advance the trial.

**Reply by MatthieuFra** [reply]
Hi, 
Thanks for the very quick response ! 

Okay, so that's what I thought. I hope my few JS skills will allow me to do so ! 

The only issue with using the audio-slider-response is that I need to use two likert scale simultaneously, like it's possible with the likert plugin. Is it easy to add more sliders to the audio-slider-response ?

**Reply by becky-gilbert** [reply]
Hi @MatthieuFra, someone else has made a multiple slider version of the html-slider-response plugin: https://github.com/jspsych/jsPsych/pull/1306 So you could incorporate those same code changes into the audio-slider-response plugin. To view the changes, you can go to the "Files changed" tab in the pull request.

And if you do decide to make an audio multiple slider plugin, please consider adding it to the [jspsych-contrib repository](https://github.com/jspsych/jspsych-contrib) so that others can use it 🙂

Let us know.if you need any further guidance!
